### ISSUE TITLE
[Feature Request] config : url

### ISSUE THREAD

**Issue by apowers313** [balderdashy/sails-redis#77]
Issue #18 was closed, but looking at [lib/connection.js](https://github.com/balderdashy/sails-redis/blob/master/lib/connection.js) it looks like it never got implemented.

This is to re-open the suggestion that the config parameters support a URL, similar to [sails-mongo](https://www.npmjs.com/package/sails-mongo#what-about-mongo-urls).

If pull requests are welcome, I would be happy to contribute.

**Comment by Ryanc1256** [comment]
@apowers313 yeah if you want to help out go a head, would help out a lot!

**Comment by sailsbot** [comment]
Thanks for posting, @apowers313.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
